ACPs/83-dynamic-multidimensional-fees/README.md

```diff
@@ -24,18 +23,20 @@ This makes transaction issuance predictable, but does not provide feedback mecha
 
 However, unlike the C-Chain, we propose a multidimensional fee scheme with an exponential update rule for each fee dimension. The [HyperSDK](https://github.com/ava-labs/hypersdk) already utilizes a multidimensional fee scheme with optional priority fees and its efficiency is backed by [academic research](https://arxiv.org/abs/2208.07919).
 
+Finally we will split the fees into two parts: there will be a `base fee`, calculated by the network and updated block by block, which must be paid by transactions to be included in a block. Moreover there will be an optional `priority fee` to be paid on top of the `base fee` which may be included by the transaction issuer to speed up inclusion.
+
 ## Specification
 
 We introduce the multidimensional scheme first and then how to apply the dynamic fee update rule for each fee dimension. Finally we list the new block verification rules, valid once the new fee scheme activates.
 
 ### Multidimensional scheme components
 
-We define four fee dimensions, `Bandwidth`, `UXTOsReads`, `UTXOsWrites`, `Compute`, to describe transactions complexity. In more details:
+We define four fee dimensions, `Bandwidth`, `DBReads`, `DBWrites`, `Compute`, to describe transactions complexity. In more details:
 
-- `Bandwidth` measures the transaction size in bytes, a proxy for the network resources needed to disseminate the transaction.
-- `UXTOsReads` measures the number of UTXOs referenced as inputs of the transactions, a proxy for the chain state reads needed to verify the transaction.
-- `UTXOsWrites` measures the number of UTXOs generated as outputs of the transactions, a proxy for the chain state writes needed to store the transaction.
-- `Compute` meters the complexity of signatures verifications, both UTXOs ones and those related to authorization of specific operations.
+- `Bandwidth` measures the transaction size in bytes, as encoded by the Avalanchego codec. Byte length is a proxy for the network resources needed to disseminate the transaction.
+- `DBReads` measures the number of DB reads needed to verify the transactions. DB reads include UTXOs reads and any other state quantity relevant for the specific transaction.
+- `DBWrites` measures the number of DB writes following the transaction verification. DB writes include UTXOs generated as outputs of the transactions and any other state quantity relevant for the specific transaction.
+- `Compute` measures the number of signatures to be verified, including UTXOs ones and those related to authorization of specific operations.
```

Comment:
Could we simplify these? The explanations seem longer than necessary.

Reply:
This idea is to try and be precise, so to let users replicate the measurements I did in the complexities repo, should they want to.
I think the details were asked by @dhrubabasu and I'll pull him in to ask his opinion